omc3/hole_in_one.py

```diff
@@ -204,6 +204,11 @@ def hole_in_one_entrypoint(opt, rest):
         Flags: **--window**
         Choices: ``('rectangle', 'welch', 'triangle', 'hann', 'hamming', 'nuttal3', 'nuttal4')``
         Default: ``hann``
+      - **resonances** *(int)*: Maximum magnet order of resonance lines to calculate.
+
+        Flags: **--resonances**
+        Choices: ``(n>=2)``
+        Default: ``4``
```

Comment:
maybe add which notation is used (European b1 = dipole, American/MAD-X convention b0 = dipole)

Reply:
and Choices: ( 6 >= n >= 2)

Reply:
Voilà